erpnext/selling/doctype/sales_order/sales_order.py

```diff
@@ -602,7 +602,12 @@ def update_item(source, target, source_parent):
 			},
 			"Sales Order Item": {
 				"doctype": "Material Request Item",
-				"field_map": {"name": "sales_order_item", "parent": "sales_order"},
+				"field_map": {
+					"name": "sales_order_item",
+					"parent": "sales_order",
+					"base_rate": "rate",
```

Comment:
Sales Rate and Purchase Rate won't be the same, maybe you can not mapped the rate fields from sales order to MR. Let the rate on the MR will be fetched from the buying price list.

Reply:
Currently, the system is not fetching the rate from buying price list. Will add the fix for that.